strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generation/NpmArtifactGeneratorTest.java

```diff
@@ -61,5 +76,76 @@ void testArtifactGeneration(@NpmRepository(repositoryId = REPOSITORY_RELEASES)
 
         // File size
         assertThat(Files.size(path)).isGreaterThan(2048);
+        
+      //License checks
+        checkLicenses(path);
     }
+    
+    public void checkLicenses(Path artifactPath) 
+            throws FileNotFoundException, IOException 
+     {
+       //1) Check that the license are located in the expected locations
+         
```

Comment:
Cosmetic: Remove extra space.
```suggestion
    public void checkLicenses(Path artifactPath) 
            throws FileNotFoundException, IOException 
    {
       //1) Check that the license are located in the expected locations
         
```